### ISSUE TITLE
Get the hash from an image?

### ISSUE THREAD

**Issue by RoyalCoder88** [fluttercommunity/flutter_blurhash#50]
Hi friends,

I have an issue with **flutter_blurhash**, how to get the hash code from an image in Flutter? 
I can't find any details for this :(

Thanks in advance!

**Comment by SoftWyer** [comment]
This library doesn't appear to support encoding an image, only decoding. The [blurhash library](https://pub.dev/packages/blurhash) should help as it has an [encode method](https://pub.dev/documentation/blurhash/latest/blurhash/BlurHash/encode.html).

**Comment by Robert-Scub** [comment]
Yes I went with a light widget as much of the same encoding as to be performed in a very efficient way and at scale where original C implementations performs best.

**Comment by dhikshith12** [comment]
Hi, @RoyalCoder88 

I have created this package [blurhash_ffi](https://pub.dev/packages/blurhash_ffi), a fast & compact version of both flutter_blurhash and blurhash_dart, it has both encoding and decoding which runs on a separate isolate with native C bindings found in original [blurha.sh](https://blurha.sh) implementation.

use that if you're interested.

**Comment by zs-dima** [comment]
@dhikshith12 thanks, although it does not support Web
